### ISSUE TITLE
(suggestion) Unable to leave cameras 

### ISSUE THREAD

**Issue by COMICLER** [tsunamayo/Starship-EVO#4827]
currently we are unable to leave a ships cameras without fully releasing ship control, i suggest just adding a leave camera feature to the ship control menu so we dont have to fully leave the ship when ever we want to switch/turn off cameras

**Comment by cloudrambler** [comment]
If there is going to be camera related options in the helm context menu, then the ultimate option would be for each linked camera to have a menu listing, along with a default listing for the native character view. That way a camera view could also change to another (named) camera view or revert to the character view, all without yielding ship control.

**Comment by COMICLER** [comment]
To be honest im not a big fan of having all cameras listed on the helm menu for a couple reasons, 1 it gets rid of one the uses for the already rarely used screens, 2 it makes it impossible to have isolated cameras for like cctv, but what would be a good alternative is to have them able to connect to hotkey gates as well as the helm menu, and to clarify i think it would be best to only have the camera you activated at the time connected to the helm menu

**Comment by cloudrambler** [comment]
I'm just thinking back to playing Elite, when ALL game interactions were screens that you cycled through. So my idea is more based on nostalgia than any real function.

I agree that having game control functions tightly linked to the physical devices has got to be the highest priority, as there is presently not yet enough interactive content to sustain immersive gameplay (bring on the coffee machines and cosy bunk beds).

**Comment by pinesh** [comment]
> currently we are unable to leave a ships cameras without fully releasing ship control, i suggest just adding a leave camera feature to the ship control menu so we dont have to fully leave the ship when ever we want to switch/turn off cameras

You can attach a camera to a screen allowing you to enter/leave cameras while in ship control. Linking a camera directly to controls is an all-or-nothing game.

**Comment by COMICLER** [comment]
No, currently you cant leave cameras anymore without leaving the whole ship

**Comment by pinesh** [comment]
> No, currently you cant leave cameras anymore without leaving the whole ship

@COMICLER  I've just built a ship to verify this, and it's sorta true but not quite. your keybindings for the ship context menu and activate are both set to the same Key, change one of them to something else and you can exit the camera while still flying the ship as normal.

**Comment by BigBadKangaroo** [comment]
@pinesh I think many players don't even know stuff like that / having two functions set on one key is nothing to be expected? ... maybe the default setting for this should change so you don't have to change key bindings to get a "good" experience. 

One thing to get rid of this situation might be to give the UI (while in camera) a marker at the bottom with the key to specifically leave just the camera?

**Comment by pinesh** [comment]
@BigBadKangaroo  Totally, It was more to verify if the ability to leave cameras was bugged or not. It doesn't appear to be bugged. Instead, the priority order for interactions has gone to ship control rather than activation. 

It's also noted, however, that you can still leave a ship control by holding down interact without ever leaving the ship menu. That's potentially an issue.

**Comment by tsunamayo** [comment]
> @BigBadKangaroo Totally, It was more to verify if the ability to leave cameras was bugged or not. It doesn't appear to be bugged. Instead, the priority order for interactions has gone to ship control rather than activation.
> 
> It's also noted, however, that you can still leave a ship control by holding down interact without ever leaving the ship menu. That's potentially an issue.

Hi, yep I think it is the correct way of doing this. Thanks
